domains/terraegg.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Portfolio for TerraEgg",
+  "owner": {
+    "username": "TerraEgg",
+    "email": "albumanyt@gmail.com"
+  },
+  "record": {
+    "CNAME": "ns1.byet.org"
```

Comment:
for ns you need to  use NS records and have a valid reason to require them

Reply:
Alright, I'll work on it thanks 👍